docs/learn/learn-account-balances.md

```diff
@@ -3,115 +3,123 @@ title: Account Balances
-Frozen : 80 DOT
-Onhold: 0 DOT
-Spendable: 20 DOT
-Untouchable: 80 DOT
+Free: 40 DOT
+Frozen (locked) : 0 DOT
+Reserved (held): 60 DOT
+Spendable: 39 DOT (Free - ED)
+Untouchable: 1 DOT (ED)
 ```
 
 ![balance-example-2](../assets/balance-example-2.png)
 
-The spendable balance would be 20 DOT (which would also include fees for future transactions from
+The spendable balance would be 39 DOT (which would also include fees for future transactions from
 this account).
 
-Note how the account cannot be reaped from the state while it has a frozen balance, or in general
+Note how the account cannot be reaped from the state while it has a reserved balance, or in general,
 any [consumer and provider reference](./learn-guides-accounts.md#query-account-data-in-polkadot-js).
 Those references determine if an account can be reaped, usually because other accounts depend on the
-existence of such an account). For example, the existential deposit adds a provider reference simply
+existence of such an account. For example, the existential deposit adds a provider reference simply
 because the account exists, while a proxy account adds a consumer reference (the proxy existence
 depends on the proxied account; the proxy is the consumer). **Because the existential deposit is
 part of the untouchable balance, the user can use all the spendable balance (there is no need to
 keep 1 DOT as spendable).**
 
-!!!info
-    The use of the _free_ balance as shown in the following figures will be possible once the _fungible_ trait is implemented for account balances.
 
-If the account creates a proxy, it will use the `free` balance as shown below.
+If the account creates a proxy, it will use the reserved balance as follows:
 
 ```
-Free: 80 DOT
-Frozen : 80 DOT
-Onhold: 20 DOT
-Spendable: 20 DOT
-Untouchable: 60 DOT
+Free: 20 DOT
+Frozen (locked) : 0 DOT
+Reserved (held): 80 DOT
+Spendable: 19 DOT (Free - ED)
+Untouchable: 1 DOT (ED)
 ```
 
 ![balance-example-3](../assets/balance-example-3.png)
 
-**Note how, through the fungible trait, the system uses the `balance` that is frozen instead of the
-`free` balance that is spendable (present configuration on-chain).** In other words, holds are
-subtracted from free balance but overlap with the frozen balance. The free portion shrinks from 100
-to 80 DOT, and the `on_hold` portion increases from 0 to 20 DOT. The creation of an identity will
-grow the `on_hold` portion to 40 DOT, and shrink further the `free` from 80 to 60 DOT. Note how the
-spendable balance stays the same in the process.
+The reserved balance for the proxy deposit increases the total held balance to 80 DOT while the free balance decreases to 20 DOT.
+
+If the account votes for a governance proposal with 20 DOT, the situation would be as follows:
 
 ```
-Free: 60 DOT
-Frozen: 80 DOT
-Onhold: 40 DOT
-Spendable: 20 DOT
-Untouchable: 40 DOT
+Free: 20 DOT
+Frozen (locked): 20 DOT
+Reserved (held): 80 DOT
+Spendable: 19 DOT (Free - ED)
+Untouchable: 21 DOT (ED)
```

Comment:
Untouchable will still be 1 DOT (just the ED) since it's "the frozen balance **in excess**" of any holds"